### PR TITLE
MULTISITE-21440: [PIPELINE] Create drush commands to connect to diffy service

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -21,13 +21,13 @@ workspace:
 matrix:
   PHP_VERSION:
     - 5.6
-    - 7.1
+    # - 7.1
```
Comment (jonhy81) [inline .drone.yml]: This file need to be removed or aligned with our drone. Feel free to remove NE Platform 3 builds and include tests to all new drush commands in pipeline (diffy and module eval)

File: includes/drush/custom/diffy.drush.inc [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function diffy_drush_command() {
+  $items = [];
+
+  // Command to refresh Diffy token based on the env DIFFY_API_KEY variable.
+  $items['diffy-refresh-token'] = [
```
Comment (jonhy81) [inline includes/drush/custom/diffy.drush.inc:15]: Will we need to setup this DIFFY API KEY into drone server?

File: includes/drush/custom/diffy.drush.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+This command requires the **DIFFY_API_KEY** environment variable to be set. If
+set correctly this command will save a Diffy token inside of the database under
+the Drupal variable name: **diffy_token**. This token will be used for
+subsequently executed Diffy commands.
+
+
+### diffy-project-snapshot
+
+This command allows you to take a snapshot of the current website IF the
+**diffy_project_id** Drupal variable is defined. If not you need to provide it
```
Comment (jonhy81) [inline includes/drush/custom/diffy.drush.md:23]: What is the plan to align the projects with this value?

**Review by jonhy81 — CHANGES_REQUESTED** [review]

File: includes/drush/custom/diffy.drush.inc (hunk 2 above)
Comment (ghost) [reply includes/drush/custom/diffy.drush.inc:15]: Exactly. And the key is used to generate the token. Which expires daily.

File: includes/drush/custom/diffy.drush.md (hunk 3 above)
Comment (ghost) [reply includes/drush/custom/diffy.drush.md:23]: I'm not sure. We should have something that's maintainable. But this is a problem without a real inventory.

I could make an extra target that creates the project and sets the project id if no id is provided.

Comment (jonhy81) [reply includes/drush/custom/diffy.drush.md:23]: Maybe we should start think about create our dinamic Inventory using the GitHub API and topics information.

File: .drone.yml (hunk 1 above)
Comment (ghost) [reply .drone.yml]: Done, but for some reason now GIT is not able to checkout the pull request any longer :(
https://drone.fpfis.eu/ec-europa/toolkit/1178/9

Comment (jonhy81) [reply .drone.yml]: Maybe you only need to sync back 3.0.22 into your branch.

**Comment by ghost** [comment]
Closing pull request because Drone does not seem to be able to checkout this PR. Will re-create branch with a clean commit.

**Pull request closed without merging**